installers/container-sirepo-base/radiasoft-download.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I think it is worth describing what this module is for

Reply:
26febff